Thanks. Can you please fix this (`npm run lint`):

```bash
~/Projects/doca-bootstrap-theme: Relequestual-patch-1 npm run lint

> doca-bootstrap-theme@0.2.0 lint /Users/miksu/Projects/doca-bootstrap-theme
> eslint components || true


/Users/miksu/Projects/doca-bootstrap-theme/components/objectDefinitionTable.js
  32:1  error  Line 32 exceeds the maximum line length of 100  max-len

✖ 1 problem (1 error, 0 warnings)
```

--- a/components/objectDefinitionTable.js
+++ b/components/objectDefinitionTable.js
@@ -1,6 +1,7 @@
 const React = require('react');
 const Constraints = require('./constraints');
 const MarkdownPreview = require('react-marked-markdown').MarkdownPreview;
+const List = require('immutable').List;
 const ImmutablePropTypes = require('react-immutable-proptypes');
 const Component = require('react-pure-render/component');
 const Definition = require('./definition');
@@ -28,7 +29,7 @@ class ObjectDefinitionTable extends Component {
               <td>
                 <p>
                   <strong>{key}</strong><br />
-                  <small><em>{definition.get('type')}</em></small>
+                  <small><em>{List.isList(definition.get('type')) ? definition.get('type').valueSeq().join(', ') : definition.get('type')}</em></small>
                 </p>
               </td>
               <td>

--- a/components/objectDefinitionTable.js
+++ b/components/objectDefinitionTable.js
@@ -29,8 +29,8 @@ class ObjectDefinitionTable extends Component {
               <td>
                 <p>
                   <strong>{key}</strong><br />
-                  <small><em>{List.isList(definition.get('type')) ? 
-                  definition.get('type').valueSeq().join(', ') : 
+                  <small><em>{List.isList(definition.get('type')) ?
+                  definition.get('type').valueSeq().join(', ') :
                   definition.get('type')}</em></small>
                 </p>
               </td>

--- a/components/objectDefinitionTable.js
+++ b/components/objectDefinitionTable.js
@@ -29,7 +29,9 @@ class ObjectDefinitionTable extends Component {
               <td>
                 <p>
                   <strong>{key}</strong><br />
-                  <small><em>{List.isList(definition.get('type')) ? definition.get('type').valueSeq().join(', ') : definition.get('type')}</em></small>
+                  <small><em>{List.isList(definition.get('type')) ? 
+                  definition.get('type').valueSeq().join(', ') : 
+                  definition.get('type')}</em></small>
                 </p>
               </td>
               <td>

